packages/api-nest/src/database/seeds/library-items.seed.ts

```
@@ -138,12 +125,15 @@ export async function seedLibraryItems(
       console.error(`[Seed] Error message: ${error?.message}`)
       if (error?.detail) console.error(`[Seed] Error detail: ${error.detail}`)
-      if (error?.constraint) console.error(`[Seed] Constraint: ${error.constraint}`)
+      if (error?.constraint)
+        console.error(`[Seed] Constraint: ${error.constraint}`)
       console.error(`[Seed] Full error:`, error)
     }
   }
 
-  console.log(`✅ [Seed] Completed: ${createdItems.length}/${sampleItems.length} items created for user ${userId}`)
+  console.log(
+    `✅ [Seed] Completed: ${createdItems.length}/${sampleItems.length} items created for user ${userId}`,
```

ESLint

[error] Delete `,`

(prettier/prettier)